frontend/src/permissions.js

```diff
@@ -13,4 +13,35 @@ const isAdmin = (user) => {
   ) !== undefined;
 };
 
+/**
```

Comment:
If we rewrite the report component to use a region passed in we may not need this function.

Reply:
True for the report component. We could still use this function for filtering the list of reports.

Reply:
Keeping the function the around even though it's not being used at the moment